tests/FunctionalJavascript/TranslationImporterTest.php

```diff
@@ -29,7 +29,7 @@ public function testTranslationImportBatch(): void {
     $this->drupalGet('import-translations');
     // Wait for the AJAX to complete. This can take a while until the batch
     // process finishes hence the increase in timeout.
-    $this->assertSession()->assertWaitOnAjaxRequest(100000);
+    $this->assertSession()->assertWaitOnAjaxRequest(1000000);
```

Comment:
Batch is faster now, do we still need such an high value? Also this is out of scope but ok if we need to fix this.

Reply:
reverted

Reply:
I have reverted value.